Snowflake.Data/Core/Tools/EnvironmentOperations.cs

```diff
@@ -19,5 +20,11 @@ public virtual string GetFolderPath(Environment.SpecialFolder folder)
         {
             return Environment.GetFolderPath(folder);
         }
+
+        public virtual string GetExecutionDirectory()
+        {
+            var executablePath = Environment.GetCommandLineArgs()[0];
```

Comment:
According to MS "The program file name can, but is not required to, include path information."
but in that case we will default to home directory, am I right @sfc-gh-knozderko ?

Reply:
If GetCommandLineArgs returns empty string, driver directory will not be used.
If GetCommandLineArgs returns executable name without path, driver directory will not be used.